index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="desktop-nav">
+        <div class="menu-icon">
+            <img src="images/ic_menu.png" alt="">
+        </div>
+        <div class="logo-mobile">
+            <img src="images/image_geometry_m_index.png" alt="">
+        </div>
+        <ul class="menu-items">
+            <li><a href="index.html">Hello</a></li>
+            <li><a href="Portfolio">Portfolio</a></li>
+            <li><a href="About">About</a></li>
+            <li><a href="Contact">Contact</a></li>
+        </ul>
+    </nav>
+    <header class="webcontainer">
+        <div class="left">
+            <h1>Leo de Andrade</h1>
+            <br><br>
+            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </div>
+        <br>
+        <div class="retangule">
+            <p>Start Collaboration</p>
+        </div>
+    </header>
+    <div class="imgindexmobile">
+        <img src="images/image_geometry_2_mobile.png" alt="">
+    </div>
+    <aside>
+        <div class="right">
+            <img src="images/image_geometry_1_1.png" alt="image_geometry_1_1.png">
+        </div>
+        <div class="dots">
+            <img src="images/image_dots.png" alt="image_dots.png">
```

Comment:
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)
The button (if the template has one) is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)

Reply:
I believe this is the next step, now it's just the skeleton. I took some advantage and write more information to rush my work.

Reply:
I am sorry but that is a requirement